uktrade/data-hub-frontend
# Add detailed header to opportunity details page

## Description of change

This PR creates a new generic component `LocalHeaderDetails` which adds a row of data to a `LocalHeader` component. 

This is then used to create the local header for the opportunity details page, which loads details of an opportunity into the page header after the task to get this data from the API succeeds (same task which fetches data for the whole page).


## Test instructions

_What should I see?_

- Go to `/investments/opportunities` and pick any opportunity
- The detailed header should appear on the page.

To test in storybook

- Go to the LocalHeaderDetails section
- There should be an example usage of the component alongside a README for it

## Screenshots
### Before
![Screenshot 2021-07-22 at 12 39 41](https://user-images.githubusercontent.com/22460823/126633541-4f36a86c-cd09-410a-8a2b-c5ed4403766c.png)


### After
![Screenshot 2021-07-22 at 12 40 16](https://user-images.githubusercontent.com/22460823/126633579-5f53d48f-96b5-469e-8bf5-84f5a4702ad9.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
